src/main/java/org/deepsymmetry/beatlink/data/ArtFinder.java

```diff
@@ -236,35 +237,7 @@ public AlbumArt getLatestArtFor(DeviceUpdate update) {
      * art around even for tracks that are not currently loaded, to save on having to request it again when another
      * track from the same album is loaded.
      */
-    private final ConcurrentLinkedHashMap<DataReference, AlbumArt> artCache =
```

Comment:
It doesn't look like you can resize the cache or introspect the cache size with Caffeine

Reply:
Please look at [Cache.policy()](https://github.com/ben-manes/caffeine/wiki/Policy) for feature-specific extensions like changing the maximum size.

Reply:
Thanks! Updated to retain the size helpers